var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -410,6 +410,8 @@ def determine_b2_options(self, spec, options):
             # change into boost compilation
             if spec.variants['cxxstd'].value == '11':
                 cxxflags.append('-std=c++11')
+        elif spec.satisfies('%fj'):
```

Comment:
Would it be better to move this above so we have:
```python
if spec.satisfies('%clang') or spec.satisfies('%fj'):
```

Reply:
fixed.